pkgs/development/python-modules/mwoauth/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib
 , buildPythonPackage
 , six
-, pyjwt
+, pyjwt1
```

Comment:
https://github.com/mediawiki-utilities/python-mwoauth/pull/43 is mentioning that PyJWT > 2 works.

Reply:
Oh, looks like **you** fixed it already @fabaff https://github.com/NixOS/nixpkgs/pull/121891 - so I reverted my change.